### ISSUE TITLE
Add Flexbox and, probably, grid lesson

### ISSUE THREAD

**Issue by glebgubckin** [hexlet-basics/exercises-css#129]
Может реализовать как отдельный отдел или просто урок в разном про flexbox, сейчас знание flex достаточно важное для новичков, а grid css было бы тоже неплохо добавить для разнообразия

**Comment by ssssank** [comment]
@Psixodelik что думаешь?

**Comment by Psixodelik** [comment]
Да, уже даже есть такие уроки. Перенесу их в основной раздел CSS
